make_UPD_igv.py

```diff
@@ -0,0 +1,176 @@
+                label = '' 
+                score = ''
+                chrom = str(position.split("_")[0])
+                if chrom != chromosome:
+                    start = 0  
+                    chromosome = chrom
+
+                snv_pos = int(position.split("_")[1])
+                start = start
+                end = snv_pos
+
+                genotype_conversion = {"0/0":"homref","0/1":"het", "1/1":"homvar"}
+
+                genotype_score = {
+                    "homref_homvar_homref": ["patIso", 1],
+                    "homref_homvar_homvar": ["matIso", -1],
+                    "homref_homvar_het": ["normal", -0.2],
+                    "homvar_homref_homvar": ["patIso", 1],
+                    "homvar_homref_homref": ["matIso", -1],
+                    "homvar_homref_het": ["normal", 0.2],
+                    "homvar_het_homref": ["matIso", -1],
+                    "het_homvar_homref": ["patIso", 1],
+                    "het_homref_het": ["patHet", 1],
+                    "homref_het_het": ["matHet", -1],
+                    "het_homref_homvar": ["patIso", 1], 
+                    "homref_het_homvat": ["matIso", -1]
+                    }
+
+                if father_geno in genotype_conversion and mother_geno in genotype_conversion and child_geno in genotype_conversion:
+                    genotype = "{}_{}_{}".format(genotype_conversion[father_geno], genotype_conversion[mother_geno],genotype_conversion[child_geno])
+                    if genotype in genotype_score:
+                        label = genotype_score[genotype][0]
+                        score = genotype_score[genotype][1]
+
+                binsize = 0
+            
+                if int(end - start) > args.maxlocus:
+                    start = end - args.maxlocus
+
+                if label: 
+                    tag = "Name={label};Position={chromosome}:{postion};Child={child_geno};Father={father_geno},Mother={mother_geno}".format(
+                        label=label,
+                        chromosome=chrom,
+                        postion=snv_pos, 
+                        child_geno=child_geno,
+                        father_geno=father_geno,
+                        mother_geno=mother_geno
+                        )
+
+                    output_file.write("{chrom}\t{start}\t{end}\t{tag}\t{score}\n".format(
+                        chrom=chrom,
+                        start=start-binsize,
+                        end=end+binsize,
+                        tag=tag,
+                        score=score
+                        ))
+
+                    start = snv_pos
+                   
+
+        output_file.close()
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('ped_file', type=argparse.FileType('r'), help='PED file')
+    parser.add_argument('run_id', help='run ID prefix')
+    parser.add_argument('sample_id', help='sample id to be processed')
```

Comment:
See also my comment in UMCUGenetics/Dx_BAF#1 I think you can make shorter argument parsers + big arguments in one go. like -i or --input_files (you can use both). then the command will be a lot shorter and more easy to use?

Reply:
Being positional arguments it's even shorter now ;)
Decided to leave args as is.